experimental/CellCulture/mESC/Differentiation/NpcDifferentiation.md

```diff
@@ -0,0 +1,22 @@
+# Mouse ES cell differentiation into NPCs
+
+1. harvest mouse ES cells by trypsinization
```

Comment:
any special plates or link to geletin or is it in the table below? How to make the gelatin?